README.md

```diff
@@ -16,6 +16,29 @@ We use [rust-clippy](https://github.com/rust-lang-nursery/rust-clippy) linters t
 
 There are plenty of [IDEs](https://areweideyet.com) and other [Rust development tools to consider](https://github.com/rust-unofficial/awesome-rust#development-tools).
 
+Currently, the library provides TCP as the main transport variant. Other data structures such as the Peer, PeerId, 
```

Comment:
This crate provides TCP-based implementation of [`libtransport`](https://github.com/Fantom-foundation/libtransport).